cmd/cli/kubectl-kyverno/log/log.go

```diff
@@ -17,9 +18,13 @@ func Configure() error {
 
 func configure(args ...string) error {
 	logging.InitFlags(nil)
+
 	if isVerbose(args...) {
-		return logging.Setup(logging.TextFormat, logging.DefaultTime, 0)
+		if level, err := getLogLevel(args...); err == nil {
```

Comment:
How is the error handled?

Reply:
`getLogLevel ` function is returning both level as well as error, so in case if error is not nil then this `configure` function will return nil.

Reply:
i think, it may be good to add an else statement to deal with error.

Reply:
@coder12git - are you working on handling the error properly?

Reply:
Yes I am, was busy with some stuff but will push my code ASAP.

Reply:
@realshuting i have added an else statement to handle error, please check